presto-kafka/src/main/java/io/prestosql/plugin/kafka/schema/ContentSchemaReader.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.kafka.schema;
+
+import io.prestosql.plugin.kafka.KafkaTableHandle;
+
+import java.util.Optional;
+
+public interface ContentSchemaReader
```

Comment:
```suggestion
public interface SchemaReader
```

Reply:
@kokosing had a comment suggesting to be more specific, i.e. to differentiate between schema for a table, a record, a database schema, etc.